### ISSUE TITLE
Gnirehtet cant provide internet access to few apps like youtube vanced and talkyou still now !!

### ISSUE THREAD

**Issue by ReckonCrusat** [Genymobile/gnirehtet#347]
//First of thnxx alot to the owner & developer of this awesome tool,its helped me alot via diff means// 
Dont know the exact reason why i am getting particular errors like this & unable to work internet around it via reverse thethering,i may guess these apps requires https connections.
Anybody have any ideas how to solve this or we can expect some solutions in coming built.

**Comment by rom1v** [comment]
Probably because the connections are blocked on your local network? Is there a firewall/proxy or something?

**Comment by ReckonCrusat** [comment]
No i didnt think so ,i dont have anything like this for android phone..
On other hand can you help me with a problem i faced with sharing internet with another smartphone( miui )device:
i get this error code mentioned below-

// Performing Streamed Install
Success
2020-12-20 21:10:31.594 INFO Main: Starting client...
Starting: Intent { act=com.genymobile.gnirehtet.START cmp=com.genymobile.gnirehtet/.GnirehtetActivity }
Security exception: Permission Denial: starting Intent { act=com.genymobile.gnirehtet.START flg=0x10000000 cmp=com.genymobile.gnirehtet/.GnirehtetActivity } from null (pid=12663, uid=2000) requires android.permission.WRITE_SECURE_SETTINGS

java.lang.SecurityException: Permission Denial: starting Intent { act=com.genymobile.gnirehtet.START flg=0x10000000 cmp=com.genymobile.gnirehtet/.GnirehtetActivity } from null (pid=12663, uid=2000) requires android.permission.WRITE_SECURE_SETTINGS
        at com.android.server.wm.ActivityStackSupervisor.checkStartAnyActivityPermission(ActivityStackSupervisor.java:1089)
        at com.android.server.wm.ActivityStarter.startActivity(ActivityStarter.java:789)
        at com.android.server.wm.ActivityStarter.startActivity(ActivityStarter.java:588)
        at com.android.server.wm.ActivityStarter.startActivityMayWait(ActivityStarter.java:1392)
        at com.android.server.wm.ActivityStarter.execute(ActivityStarter.java:519)
        at com.android.server.wm.ActivityTaskManagerService.startActivityAsUser(ActivityTaskManagerService.java:1091)
        at com.android.server.wm.ActivityTaskManagerService.startActivityAsUser(ActivityTaskManagerService.java:1065)
        at com.android.server.am.ActivityManagerService.startActivityAsUser(ActivityManagerService.java:3569)
        at com.android.server.am.ActivityManagerShellCommand.runStartActivity(ActivityManagerShellCommand.java:518)
        at com.android.server.am.ActivityManagerShellCommand.onCommand(ActivityManagerShellCommand.java:172)
        at android.os.ShellCommand.exec(ShellCommand.java:104)
        at com.android.server.am.ActivityManagerService.onShellCommand(ActivityManagerService.java:10089)
        at android.os.Binder.shellCommand(Binder.java:881)
        at android.os.Binder.onTransact(Binder.java:765)
        at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:4727)
        at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:2799)
        at android.os.Binder.execTransactInternal(Binder.java:1021)
        at android.os.Binder.execTransact(Binder.java:994)
2020-12-20 21:10:32.112 ERROR Main: Cannot start client: Command adb ["shell", "am", "start", "-a", "com.genymobile.gnirehtet.START", "-n", "com.genymobile.gnirehtet/.GnirehtetActivity"] returned with value 255//

**Comment by rom1v** [comment]
See #5

**Comment by ReckonCrusat** [comment]
friend if  you dont mind i am not a techie guy ,so didnt understand a single thing regarding the codes parts.
can you tell me in layman's steps required by me to bypass this issue like-what codes should i execute..
or from what i understand till now ,miui is a very bad rom yet so didnt giving permission to execute commands via adb..but how to solve this issue

**Comment by CDJenson** [comment]
> miui is a very bad rom yet so didnt giving permission to execute commands via adb
不是我不行，是它太烂👏👏👏。

**Comment by ReckonCrusat** [comment]
> > miui is a very bad rom yet so didnt giving permission to execute commands via adb
> > 不是我不行，是它太烂👏👏👏。
i know that part miui being stubborn and dont letting the exe run properly,but there must be a solution around to bypass the restrictions,right?

**Comment by CDJenson** [comment]
> > > miui is a very bad rom yet so didnt giving permission to execute commands via adb
> > > 不是我不行，是它太烂clapclapclap。
> > > i know that part miui being stubborn and dont letting the exe run properly,but there must be a solution around to bypass the restrictions,right?

you know a piece of shit,but BB is your freedom.🤟

**Comment by ReckonCrusat** [comment]
> > > > miui is a very bad rom yet so didnt giving permission to execute commands via adb
> > > > 不是我不行，是它太烂clapclapclap。
> > > > i know that part miui being stubborn and dont letting the exe run properly,but there must be a solution around to bypass the restrictions,right?
> 
> you know a piece of shit,but BB is your freedom.🤟

Hey asshole !you bloody Chinese people are always there to pollute the environment (gene mutating bat and nostalgic animals),interfere with in everybody's matter and never want to see others happy & in peace...
Does ur mom told u to come here and pollute this forum too by using slangs first(If I were some moderator here I should kick away ur small ass long ago with putting 9 heart octopus *shrimp inside it that u people  eat)

Whats wrong in me saying - "Miui is literally the worst rom ever been made" ...everyone knows it & its all over the net,u better come out of your bubble Lil milk sucking child crying for lol candy...

Miui & all Chinese people always know is how to steal from others & nobody better than them...& if somebody caught them or questions them,they will go directly go to war with them or disappear them from face of earth(poor jack MA ! He always want betterment of people & some modifications in banking policies )

If u can't solve a problem don't bother to make a new one !! & fare warning to u again --don't u dare to reply me here anymore,if u do I will fuck ur mama infront of u with my words....PERIOD .🤟🤟🤟 **FUCK YOU**#$
